### ISSUE TITLE
Question about image data format

### ISSUE THREAD

**Issue by ZiqianXie** [DeepTrial/Retina-VesselNet#13]
I was looking at your dense_unet model, the input shape is (patch_width, patch_height, 1), so it is channel-last ordering, but then the batch normalization is done on axis=1, which corresponds to channel-first ordering. Is this a bug or did I miss something?

**Comment by DeepTrial** [comment]
i have to admit that it's bug.I pushed the wrong model which written in theano.thx

**Comment by ZiqianXie** [comment]
Thanks for your good work, can you also update the pretrained model?

**Comment by DeepTrial** [comment]
oh,of course! I will update as soon as possible

**Comment by DeepTrial** [comment]
I already update the whole project(including  new download-link of  pretrained model ),please check

**Comment by ZiqianXie** [comment]
I really appreciate it, thank you!
